tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -375,6 +384,40 @@ def _helper(self, kernel, dtype):
         kernel((N,), (N,), (x1, x2, y, N**2))
         return x1, x2, y
 
+    def test_compile_kernel(self):
+        kern = cupy.RawKernel(
+            _test_compile_src, 'test_op',
+            options=('-DOP=+',),
+            backend=self.backend)
+        kern.compile()
+        x1, x2, y = self._helper(kern, cupy.float32)
+        assert cupy.allclose(y, x1 + x2)
+        kern = cupy.RawKernel(
+            _test_compile_src, 'test_op',
+            options=('-DOP=*',),
+            backend=self.backend)
```

Comment:
I'd prefer to see this tested:
```suggestion
        kern.options = ('-DOP=*',)
```